qa/L0_storage_S3_local/test.sh

```diff
@@ -341,6 +342,7 @@ else
     RET=1
   fi
 fi
+export AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY_BACKUP
```

Comment:
Add comment with reasons for resetting key to valid after invalid test.

Reply:
Added: [Add reason for restoring keys](https://github.com/triton-inference-server/server/pull/5832/commits/12a5449af238d063bbf712c0539b30c53bc18410)